app/views/groups/show.html.erb

```diff
@@ -42,6 +42,9 @@
   <%= link_to 'Lock Group', lock_draw_group_path(@draw, @group), method: :put, class: 'button alert' if policy(@group).lock? %>
   <%= link_to 'Finalize Group', finalize_draw_group_path(@draw, @group), method: :put, class: 'button' if policy(@group).finalize? %>
   <%= link_to 'Finalize Membership', finalize_membership_draw_group_path(@draw, @group), method: :put, class: 'button' if policy(@group).finalize_membership? %>
+  <% if policy(@group).make_drawless? %>
+    <%= link_to 'Make special group', make_drawless_draw_group_path(@draw, @group), method: :patch, class: 'button alert' %>
```

Comment:
this should have a tooltip: "Remove this group from their draw due to special circumstances" or something